third_party/airflow/tox.ini

```diff
@@ -0,0 +1,58 @@
+[tox]
```

Comment:
Does this file mean we will need to run tox commands from two places now just to get a CI pass? 

If that is the case, do we maybe want a makefile that cn auto-run the commands for us? Just for a bit of ease of use.

Reply:
Makefiles aren’t really a common tool for python developers.  I would like to approach build systems as being native like to their language.  Also I hope some day we can maybe migrate clients to separate repos so I don’t want the go makefile controlling the build of our python code

Reply:
> Does this file mean we will need to run tox commands from two places now just to get a CI pass?
> 
> If that is the case, do we maybe want a makefile that can auto-run the commands for us? Just for a bit of ease of use.

I think this is more of a bandaid.  Our goal should be that the airflow operator uses a published python client but we haven't picked up deployment of that yet.

Reply:
It's totally reasonable to have something in the makefile to run both. It can be done as a follow on pr as desired.